criu/protobuf.c

```diff
@@ -66,6 +66,10 @@ int do_pb_read_one(struct cr_img *img, void **pobj, int type, bool eof)
 		ret = 0;
 	else
 		ret = bread(&img->_x, &size, sizeof(size));
+
+	/* This is to make coverity happy and to not report TAINTED_SCALAR. */
+	size &= UINT32_MAX;
```

Comment:
This is not a false positive, but I am not sure that we want to care about this. But I don't like this hack in the code and I think we can just ignore this issue for now.